files/justfiles/secureblue.just

```diff
@@ -730,6 +736,8 @@ dns-selector:
     readonly policy_file="/etc/chromium/policies/managed/10-securedns-browser.json"
     mkdir -p /etc/systemd/resolved.conf.d/
     mkdir -p /etc/chromium/policies/managed/
+    restorecon -R /etc/systemd/resolved.conf.d
+    restorecon -R /etc/chromium/policies/managed
```

Comment:
same as above, `restorecon -Rv /etc/chromium/policies/managed` does not report changes

Reply:
Ah yeah that's it. Secureblue provides those folders by default.